flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/util/UpdatingPlanChecker.scala

```diff
@@ -116,14 +135,100 @@ object UpdatingPlanChecker {
           val windowStartEnd = w.getWindowProperties.map(_.name)
```

Comment:
This can not only be start and end but also `.rowtime` or `.proctime`.

Reply:
Window can only output start or end field, right ?

Reply:
No, it can also emit the rowtime and proctime.